resolwe_bio/tools/star_sj_to_bed12.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/python3
+"""Recalculate from STAR SJ.out.tab file to BED12 format."""
+
+import argparse
+
+import numpy as np
+import pandas as pd
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument('-f', '--sj_file', help="STAR SJ.out.tab output file")
+args = parser.parse_args()
+
+sj_file = pd.read_csv(args.sj_file, delimiter='\t', header=None)
+
+# BED12 consists of 12 columns:
+header = ['chromosome',
+          'sj_start',
+          'sj_end',
+          'sj_name',
+          'score',
+          'strand',
+          'thick_start',
+          'thick_end',
+          'item_rgb',
+          'block_counts',
+          'block_sizes',
+          'block_starts']
+
+bed_file = pd.DataFrame(index=sj_file.index, columns=header)
+# 1: chromosome = first column from STAR SJ.out.tab
+bed_file.loc[:, 'chromosome'] = sj_file.iloc[:, 0].values
+
+# 2: SJ start (0-based) =
+# (first base of the intron (1-based) - maximum spliced alignment overhang) -1 (to recalculate to
+# 0 based system)
+bed_file.loc[:, 'sj_start'] = (sj_file.iloc[:, 1]) - (sj_file.iloc[:, 8]) - 1
+
+# 3: SJ end (0-based)
+# = (last base of the intron (1-based) + maximum spliced alignment overhang)
+bed_file.loc[:, 'sj_end'] = (sj_file.iloc[:, 2]) + (sj_file.iloc[:, 8])
+
+# 4: SJ name
+bed_file.loc[:, 'sj_name'] = 'JUNC000' + (sj_file.index + 1).astype(str)
```

Comment:
I don;t know if this is problematic bit currently this will look like so:
index = 1: `JUNC000`
index = 12: `JUNC00012`
index = 123: `JUNC000123`

i beleive it would be better to have all items of the same size:
index = 1: `JUNC000`
index = 12: `JUNC0012`
index = 123: `JUNC0123`

You can achieve this by using this example:
`'{:0>4}'.format('1')` this prints: `0001`
`'{:0>4}'.format('12')` this prints: `0012`
`'{:0>4}'.format('123')` this prints: `0132`

Reply:
Maybe ```(df.index + 1).astype(str).map(lambda x: 'JUNC' + x.zfill(3))``` or ```(df.index + 1).astype(str).map(lambda x: 'JUNC{:0>4}'.format(x))```